.platform.app.yaml

```diff
@@ -0,0 +1,92 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+
+# The name of this app. Must be unique within a project.
+name: php
+
+# The toolstack used to build the application.
+type: "php"
```

Comment:
That's the only language supported by Platform.sh at the moment. It doesn't actually matter since Sphinx builds a static HTML website.